pallene/checker.lua

```diff
@@ -636,29 +638,47 @@ function FunChecker:check_var(var)
         elseif cname._tag == "checker.Name.Function" then
             var._type = self.p.module.functions[cname.id].typ
         elseif cname._tag == "checker.Name.Builtin" then
-            var._type = builtins[cname.name].typ
+            var._type = builtins.functions[cname.name].typ
         elseif cname._tag == "checker.Name.Module" then
             -- The type for a module is a table.
-            var._type = builtin_modules[cname.name]
+           var._type = builtins.modules[cname.name]
         else
-            error("Unexpected condition: cname._tag == " .. cname._tag)
+            error("impossible")
         end
 
+        -- print(inspect(var))
```

Comment:
More print comments. Please remove all of them. (Consider using `git add -p`, which makes it possible to leave them in the text file but not include them in the commit)